packages/react/src/auto/polaris/inputs/PolarisAutoInput.tsx

```diff
@@ -46,6 +46,9 @@ export const PolarisAutoInput = (props: { field: string }) => {
       return <PolarisAutoBooleanInput field={props.field} />;
     }
     case FieldType.DateTime: {
+      // TODO: If a user sets the value of the DateTimePicker, and does not provide a custom onChange function
+      // the value will never be updated.
+      // Should an error be thrown in this case? I.e. value && !onChange => throw error
```

Comment:
NB; thoughts?
I'll remove this comment once I get feedback from reviewers

Reply:
I don't think an error is required in this case. From what I've seen, most UI libraries will allow you to pass in a value and leave the component unchangeable with that